pkg/apis/rollouts/v1alpha1/types.go

```diff
@@ -989,6 +989,8 @@ const (
 	RolloutPaused RolloutConditionType = "Paused"
 	// RolloutCompleted means that rollout is in a completed state. It is still progressing at this point.
```

Comment:
While you are here, the message `It is still progressing at this point.` is inaccurate. I'd like to change the message to: 

> RolloutCompleted indicates that the rollout completed its update to the desired revision and is not in the middle of any update. Note that a Completed rollout could also be considered Progressing or Degraded, if its Pods become unavailable sometime after the update completes.